config/recipes/driver-container-base/manifests/0000-driver-container-base-build.yaml

```diff
@@ -107,12 +107,14 @@ data:
 
     # Installation order is important leave this as is 
     kernel_rpms=(
-      $(find /extensions -name kernel-core-${UNAME}.rpm -exec ls {} \; | tail -n1)
-      $(find /extensions -name kernel-devel-${UNAME}.rpm -exec ls {} \; | tail -n1)
-      $(find /extensions -name kernel-headers-${UNAME}.rpm -exec ls {} \; | tail -n1) 
-      $(find /extensions -name kernel-modules-${UNAME}.rpm -exec ls {} \; | tail -n1)
-      $(find /extensions -name kernel-modules-extra-${UNAME}.rpm -exec ls {} \; | tail -n1)
+      $(find /extensions -name kernel-core-${kernel_version}.rpm -exec ls {} \; | tail -n1)
```

Comment:
Nice catch!

Reply:
This only fixes part of the problem though, see my slack message when you get a chance